### DISCUSSION TITLE
Demo Page 404

### DISCUSSION THREAD

**Discussion by mandud** [Laravel-Backpack/community-forum#1041] [Q&A (Help)] [answered]
Hi Everyone,

Just purchased Backpack Pro and try to git demo backpack
Found when open page getting 404 error

Check on the demo.backpackforlaravel.com also getting same error

Still didn't found where the issue, kindly help to fix it

Thanks

**Comment by jcastroa87** [comment]
Hello @mandud , what country are you from? are you using VPN? it is a home IP or a shared one?

Cheers.

**Comment by mandud** [comment]
Hi @jcastroa87 

I'm from Indonesia, I'm not using VPN and using home broadband connection

404 error seems happened with another too, I found in another post tell Laravel Backpack only provide backend, front end should be configured by us, I believe because in the demo not providing route for every page that already created 

I'm trying to open this one, then it's show 404 page
![image](https://github.com/Laravel-Backpack/community-forum/assets/3507479/4d9a933d-9c36-4d38-a407-cdb925a93441)

Hopefully demo page can providing all features works well, since I found roles and permission also not working

Thanks

**Comment by jcastroa87** [comment]
Hello @mandud now i understand your request, i think you can not access any repo or our full demo.

First, our demo is just to show features, some of them are disable because security reasons.
When you install PageManager, did you follow all the instructions in our docs? did you add the [catch-all routes](https://github.com/Laravel-Backpack/PageManager?tab=readme-ov-file#example-front-end)?

Please show us the routes code to check about it.

Thanks

**Reply by mandud** [reply]
Hi @jcastroa87 

Yeah, actually I ended with that article so currently there's no issue with showing the page

Honestly, I'm struggling use Laravel Backpack from scratch since I'm limited on basic Laravel knowledge
So, I'm trying to use the full demo to learn and rebuild the apps

Current issue the permission on the demo didn't work, hope there's some clear article to fix this issue

Thanks

**Reply by jcastroa87** [reply, marked as answer]
Again, i fully recommended not use demo to build an app, have a lot of features disabled, i recommend to use our docs to install addons, in case of permission is [this one](https://github.com/Laravel-Backpack/PermissionManager).

Cheers.
